ArduinoCode/Pedal_Box/Pedal_Box.ino

```diff
@@ -131,7 +154,8 @@ void readPotentiometers() {
     
     unsigned char message[8] = {accelTorque,regenTorque,0,0,isForward,1,0,0};
     CAN.sendMsgBuf(CAN_MOTOR, 0, 8, message); // send message to motor controller with torques and direction
-  } else {  // turn off motor and send error message
+  } else {  // send shutdown message, turn off motor, and send error message
```

Comment:
Doing BMS first means you'll be cutting off power into the inverter immediately. However given the capacitance of the inverter I theoretically think this should be fine and it'll still get shutdown normally as well anyways. Just an FYI